lib/passport-twitch/oauth2.js

```diff
@@ -72,26 +77,40 @@ util.inherits(Strategy, OAuth2Strategy);
  * @api protected
  */
 Strategy.prototype.userProfile = function(accessToken, done) {
-    this._oauth2.get("https://api.twitch.tv/kraken/user", accessToken, function (err, body, res) {
-        if (err) { return done(new InternalOAuthError("failed to fetch user profile", err)); }
-
-        try {
-            var json = JSON.parse(body);
+    console.log('userProfile', accessToken)
```

Comment:
You should remove all the console.log or add a sort of debug variable to enable these console messages.